src/core/core.scale.js

```diff
@@ -840,10 +855,14 @@ module.exports = function(Chart) {
 
 				if (isHorizontal) {
 					scaleLabelX = me.left + ((me.right - me.left) / 2); // midpoint of the width
-					scaleLabelY = options.position === 'bottom' ? me.bottom - halfLineHeight : me.top + halfLineHeight;
+					scaleLabelY = options.position === 'bottom'
+						? me.bottom - halfLineHeight - scaleLabelPadding.bottom
+						: me.top + halfLineHeight + scaleLabelPadding.top;
 				} else {
 					var isLeft = options.position === 'left';
-					scaleLabelX = isLeft ? me.left + halfLineHeight : me.right - halfLineHeight;
+					scaleLabelX = isLeft
+						? me.left + halfLineHeight + scaleLabelPadding.top
+						: me.right - halfLineHeight - scaleLabelPadding.top;
```

Comment:
[nit] would put `halfLineHeight + scaleLabeePadding.top` in a local var for better minification since it's use 3 times

Reply:
Can do but it looks kinda asymetric and hard to grasp:

				var halfLineHeight = parseLineHeight(scaleLabel) / 2;
				var halfLineHeightPlusTopPadding = halfLineHeight + scaleLabelPadding.top;

				if (isHorizontal) {
					scaleLabelX = me.left + ((me.right - me.left) / 2); // midpoint of the width
					scaleLabelY = options.position === 'bottom'
						? me.bottom - halfLineHeight - scaleLabelPadding.bottom
						: me.top + halfLineHeightPlusTopPadding;
				} else {
					var isLeft = options.position === 'left';
					scaleLabelX = isLeft
						? me.left + halfLineHeightPlusTopPadding
						: me.right - halfLineHeightPlusTopPadding;
					scaleLabelY = me.top + ((me.bottom - me.top) / 2);
					rotation = isLeft ? -0.5 * Math.PI : 0.5 * Math.PI;
				}

Is this really desired?

Reply:
fair enough, that's not better. Lets leave it as is